### DISCUSSION TITLE
Failed to initiate the GitHub login process. Please try again.

### DISCUSSION THREAD

**Discussion by q920447939** [community/community#31447] [Copilot Conversations] [answered]
before i use github copilot old version . use a special way to use the function(use replace core-1.1.2xx.jar with core-1.1.20.jar )
yesterday ,My github copilot is being asked to force an upgrade to the latest version
But it can't be used normally
hint GitHub Copilot: Failed to initiate the GitHub login process. Please try again.
i try this page top method(set IDEA HTTP proxy) and check connection (` https://copilot-proxy.githubusercontent.com/_ping`) .
IDEA show check connection is connection successful
i found other website method . set IDEA VM option (`-Dcopilot.agent.disabled=true`). then restart IDEA ,but github copilot plugin alway hint `GitHub Copilot: Failed to initiate the GitHub login process. Please try again.`
then i found the github-copilot-intellij-1.1.30.1887 dir exists copilot-agent
i try use CMD command run `copilot-agent-win.exe` in the directory bin ， but hint me
`
Node.js is only supported on Windows 8.1, Windows Server 2012 R2, or higher. Setting the NODE_SKIP_PLATFORM_CHECK environment variable to 1 skips this check, but Node.js might not execute correctly. Any issues encountered on unsupported platforms will not be fixed.
`
then i set NODE_SKIP_PLATFORM_CHECK=1 in my system environment
the copilot-agent-win.exe Can finally start. but github copilot still hine me login process.
i also try restart IDEA and clean IDEA cache .
All the methods tried in the end didn't work

IDEA info:
`
IntelliJ IDEA 2021.1 (Ultimate Edition) Build #IU-211.6693.111, built on April 6, 2021 Licensed to IntelliJ IDEA Evaluator Expiration date: September 25, 2022 Runtime version: 11.0.10+9-b1341.35 amd64 VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o. Windows 7 6.1 GC: G1 Young Generation, G1 Old Generation Memory: 3072M Cores: 6 Registry: compiler.automake.allow.when.app.running=true Non-Bundled Plugins: Key Promoter X (2021.1.1), com.huage2580.leeks (1.9.2), com.intellij.ideolog (203.0.27.0), io.zhile.research.ide-eval-resetter (2.1.6), com.github.copilot (1.1.30.1887), GrepConsole (12.2.211.6693.1), String Manipulation (8.15.203.000.3), org.jetbrains.jumpToLine (0.1.8), org.intellij.plugins.junitgen (2.0.12), MavenRunHelper (4.13.203.000.0), com.alibabacloud.intellij.toolkit-intellij (2022.5.2-211), JRebelPlugin (2021.3.0), SequenceDiagram (2.1.8), cn.yiiguxing.plugin.translate (3.2.2), com.bruce.intellijplugin.generatesetter (2.8), com.ccnode.codegenerator.MyBatisCodeHelperProMarketPlace (2.9.3-183-204) Kotlin: 211-1.4.32-release-IJ6693.72
`

system info :
`Microsoft Windows 7 (6.1) Ultimate Edition 64-bit Service Pack 1 (Build 7601) `

**I can't believe that the update of copilot to 1.1.31.1908 still doesn't solve this problem. Is it so difficult to have an HTTP proxy function?**

**Comment by davecheney** [comment, marked as answer]
Please see https://github.com/orgs/community/discussions/29127 for HTTP proxy support.

**Reply by q920447939** [reply]
I have already set up an HTTP proxy and tested the target website to return results is connection succ。

**Reply by davecheney** [reply]
Please continue the discussion on https://github.com/orgs/community/discussions/29127
